README.md

```diff
@@ -461,6 +461,13 @@ sequencer.insertSteps({
 ```
 return value: **`sequencer`** (To allow method chaining)
 
+## Meta Modules
+
+Image sequencer supports saving a sequence of modules and their associated settings(a stringified sequence) as a `meta-module`. These meta-modules are saved in the local storage inside the browser and inside a json file in node.js. Meta-modules can be saved in node context using the CLI option
```

Comment:
needs a space here, small typo!  and capitalize `Image Sequencer`? but looking great!

maybe `and their associated settings in a simple string syntax, and then wrapping them up as a new module -- a "meta module" made of other modules. These meta-modules can be saved in ... or inside a json file. `

Reply:
done!!